specification/columns/servicesubcategory.md

```diff
@@ -2,10 +2,12 @@
 
 The Service Subcategory is a secondary classification of the Service Category for a [*service*](#glossary:service) based on its core function. The Service Subcategory (in conjunction with the Service Category) is commonly used for scenarios like analyzing spend and usage for specific workload types across providers and tracking the migration of workloads across fundamentally different architectures.  
 
-* The ServiceSubcategory column MUST be present in a FOCUS dataset and MUST NOT be null.
-* This column is of type String and MUST be one of the allowed values.
-* Each Service Subcategory value MUST have one and only one Service Category parent.
-* Though a given *service* can have multiple purposes, each *service* SHOULD have one and only one Subcategory that best aligns with its primary purpose.
+The ServiceSubcategory column adheres to the following requirements:
+
+* ServiceSubcategory is RECOMMENDED to be present in a FOCUS dataset and MUST NOT be null.
+* ServiceSubcategory is of type String and MUST be one of the allowed values.
+* Each ServiceSubcategory value MUST have one and only one ServiceCategory parent.
+* Though a given *service* can have multiple purposes, each *service* SHOULD have one and only one ServiceSubcategory that best aligns with its primary purpose.
```

Comment:
**One more question:** 
What do you think about **referencing the `ServiceName` column as well, rather than just the term in the glossary**? 
Note: For `ServiceCategory`, we also referenced only the term in the glossary and not the `ServiceName` column

I’m sure there’s a better way to do it if we decide to... 

```suggestion
* Though a given *service* can have multiple purposes, each *service* ([ServiceName](#servicename) SHOULD have one and only one ServiceSubcategory that best aligns with its primary purpose.
```

Reply:
How would this work with services that depend on other services? Example Data Science and Kubernetes may depend on compute?

Reply:
I'm in favor of keeping this as is. I see the point but this is consistently the way we've done this for Charge/Billing Periods, SKUs etc. If we do want to move forward with this, I might suggest the following:

(Same change would need to be made for ServiceCategory if we're going forward with this) 

```suggestion
* Though a given *service* can have multiple purposes, each *service* (identified by columns like [ServiceName](#servicename) SHOULD have one and only one ServiceSubcategory that best aligns with its primary purpose.
```

Reply:
> How would this work with services that depend on other services? Example Data Science and Kubernetes may depend on compute?

Hey @arrarama .. the goal of ServiceCategory and Service Subcategory is to categorize based on the primary purpose of that service (regardless of the underlying services used). E.g. a relational database service is built based on compute, storage, and networking foundational services. However, its primary purpose is to serve database needs - thus it should use the database service category.

There's a separate discussion about being charged for different SKUs within the service - like compute or storage related SKUs that have pricing attached. That would eventually be captured at the SKU level (SkuCategory or something along those lines) whereas this category is for the service-level.